packages/ripple/index.tsx

```diff
@@ -325,6 +334,10 @@ export function withRipple<
       });
     };
 
+    disabledRipplePostProcessor = () => {
+      this.adapter.removeClass(cssClasses.BG_FOCUSED);
```

Comment:
We shouldn't be calling the adapter directly, which is why we didn't have it on the component. I think we can actually just call handleBlur.

Reply:
I reflected it and write some test code!